lib/internal/modules/esm/translators.js

```diff
@@ -177,48 +169,51 @@ function enrichCJSError(err) {
 // Strategy for loading a node-style CommonJS module
 const isWindows = process.platform === 'win32';
 const winSepRegEx = /\//g;
-translators.set('commonjs', async function commonjsStrategy(url, isMain) {
-  debug(`Translating CJSModule ${url}`);
-
-  let filename = internalURLModule.fileURLToPath(new URL(url));
-  if (isWindows)
-    filename = StringPrototypeReplace(filename, winSepRegEx, '\\');
-
-  if (!cjsParse) await initCJSParse();
-  const { module, exportNames } = cjsPreparseModuleExports(filename);
-  const namesWithDefault = exportNames.has('default') ?
-    [...exportNames] : ['default', ...exportNames];
-
-  return new ModuleWrap(url, undefined, namesWithDefault, function() {
-    debug(`Loading CJSModule ${url}`);
-
-    let exports;
-    if (asyncESM.ESMLoader.cjsCache.has(module)) {
-      exports = asyncESM.ESMLoader.cjsCache.get(module);
-      asyncESM.ESMLoader.cjsCache.delete(module);
-    } else {
-      try {
-        exports = CJSModule._load(filename, undefined, isMain);
-      } catch (err) {
-        enrichCJSError(err);
-        throw err;
+translators.set(
+  'commonjs',
+  async function commonjsStrategy(url, source, isMain) {
+    debug(`Translating CJSModule ${url}`);
+
+    let filename = internalURLModule.fileURLToPath(new URL(url));
+    if (isWindows)
+      filename = StringPrototypeReplace(filename, winSepRegEx, '\\');
+
+    if (!cjsParse) await initCJSParse();
+    const { module, exportNames } = cjsPreparseModuleExports(filename);
+    const namesWithDefault = exportNames.has('default') ?
+      [...exportNames] : ['default', ...exportNames];
+
+    return new ModuleWrap(url, undefined, namesWithDefault, function() {
+      debug(`Loading CJSModule ${url}`);
+
+      let exports;
+      if (asyncESM.esmLoader.cjsCache.has(module)) {
+        exports = asyncESM.esmLoader.cjsCache.get(module);
+        asyncESM.esmLoader.cjsCache.delete(module);
+      } else {
+        try {
+          exports = CJSModule._load(filename, undefined, isMain);
+        } catch (err) {
+          enrichCJSError(err);
+          throw err;
+        }
       }
-    }
 
-    for (const exportName of exportNames) {
-      if (!ObjectPrototypeHasOwnProperty(exports, exportName) ||
-          exportName === 'default')
-        continue;
-      // We might trigger a getter -> dont fail.
-      let value;
-      try {
-        value = exports[exportName];
-      } catch {}
-      this.setExport(exportName, value);
-    }
-    this.setExport('default', exports);
-  });
-});
+      for (const exportName of exportNames) {
+        if (!ObjectPrototypeHasOwnProperty(exports, exportName) ||
+            exportName === 'default')
+          continue;
+        // We might trigger a getter -> dont fail.
+        let value;
+        try {
+          value = exports[exportName];
+        } catch {}
+        this.setExport(exportName, value);
+      }
+      this.setExport('default', exports);
+    });
+  }
+);
```

Comment:
I think VS Code might've done this (reindent)

I noticed this preexisting code used iteration (`for of`). Is avoiding iteration an official policy? If so, is it documented anywhere? (This suggests it's not known)

Reply:
> I think VS Code might've done this (reindent). GitHub's "hide whitespace changes" correctly shows what was actually changed.

Kindly roll it back please, whitespace changes make `git blame` less useful and can be a real pain when backporting to LTS lines.

> I noticed this preexisting code used iteration (for of). Is avoiding iteration an official policy? If so, is it documented anywhere? (This suggests it's not known)

It's being documented in https://github.com/nodejs/node/pull/38635; iteration is not an issue in this case because `exportNames` is a `SafeSet`, not an array.